src/anomalib/utils/config.py

```diff
@@ -214,6 +217,21 @@ def _update_nncf_config(config: DictConfig | ListConfig) -> DictConfig | ListCon
     return config
 
 
+def _update_val_config(config: DictConfig | ListConfig) -> DictConfig | ListConfig:
+    """Skip validation if `val_split_mode` is set to 'none'.
+
+    Args:
+        config (DictConfig | ListConfig): Configurable parameters of the current run.
+
+    Returns:
+        DictConfig | ListConfig: Updated configurable parameters in DictConfig object.
+    """
+    if config.data.init_args.val_split_mode == ValSplitMode.NONE and config.trainer.limit_val_batches != 0.0:
```

Comment:
Are we sure that if `val_split_mode` is None, that validation dataset is not provided separately?